docs/src/main/asciidoc/spanner.adoc

```diff
@@ -420,6 +420,9 @@ If a property marked for lazy fetching is never retrieved, then it is also skipp
 
 If used inside a transaction, subsequent operations on lazily-fetched properties use the same transaction context as that of the original parent entity.
 
+===== Soft Delete
```

Comment:
Soft Delete is just one application of this. The section should probably be called something like "Declarative Filtering with `@Where`"

Reply:
ok
Should I add more information like examples? For me, it is a trivial use-case that Hirberhane has for years.

Reply:
Yeah a small example of soft-delete would be great!

Reply:
will add one